### ISSUE TITLE
NPE

### ISSUE THREAD

**Issue by sulmansarwar** [bbottema/simple-java-mail#97]
Hello,

I am using the following code snippet but receiving an NPE. I have attached the error log dow below. Any help would be appreciated. Thanks.

```
final Email emailMessage = new EmailBuilder()
                .from("Admin", "noreply@example.com")
                .to(user.getName(), user.getEmail())
                .subject("example.com account confirmation")
                .text("Please follow this link to confirm your " +
                        "account creation: \n\n" + confirmURL + " .")
                .build();

        final Mailer mailer = new Mailer(
                mailerConfig.getSmtpServer(),
                mailerConfig.getSmtpPort(),
                mailerConfig.getNoReplyUser(),
                mailerConfig.getNoReplyPassword(),
                TransportStrategy.SMTP_PLAIN
        );

        mailer.setDebug(true);
        mailer.validate(emailMessage);
        mailer.sendMail(emailMessage);
```

Error Log

```
io.dropwizard.jersey.errors.LoggingExceptionMapper: Error handling a request: abf23f0b6ace03fc
! java.lang.NullPointerException: null
! at org.simplejavamail.mailer.Mailer.validate(Mailer.java:400)
! at com.xxx.mailer.NewUserMailer.sendAccountCreationConfirmationEmail(NewUserMailer.java:49)
! at com.xxx.resources.UserRegistrationResource.registerUser(UserRegistrationResource.java:61)
```

**Comment by bbottema** [comment]
Yes it seems the last version introduced a critical error. Related to #91. Please revert back to an older version. I hope to have it sorted out by the end of today.

**Comment by sulmansarwar** [comment]
Ok. Thanks. 
Last successful version that resolves this bug is 
`<simple-java-mail>4.2.1</simple-java-mail>`

**Comment by bbottema** [comment]
Should be solved in 4.4.2, please check if you have some time (might take a few minutes to appear in Maven Central).
